### PR TITLE
Move extension for request namespace

### PR DESCRIPTION
**Pull request by inakianduaga**
Export types
Bump up version to 2
Cleanup tsconfig

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,6 +1,15 @@
-import _client from './src/client'
+import _client, { ToguruClient } from './src/client'
 import * as _expressBridge from './src/express/bridge'
 
+declare global {
```
Comment (inakianduaga) [inline index.ts]: why do we need to move the global declaration here? seems strange

File: tsconfig.json [hunk 2]
```diff
@@ -4,56 +4,16 @@
...
-        // "composite": true,                     /* Enable project compilation */
-        // "removeComments": true,                /* Do not emit comments to output. */
-        // "noEmit": true,                        /* Do not emit outputs. */
-        // "importHelpers": true,                 /* Import emit helpers from 'tslib'. */
-        // "downlevelIteration": true,            /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
-        // "isolatedModules": true,               /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
-
-        /* Strict Type-Checking Options */
+        "declaration": true,
+        "declarationDir": "./dist",
```
Comment (inakianduaga) [inline tsconfig.json]: if it's the same folder as the output, we probably don't need to specify `declarationDir`

File: tsconfig.json [hunk 3]
```diff
@@ -4,56 +4,16 @@
...
-        // "mapRoot": "",                         /* Specify the location where debugger should locate map files instead of generated locations. */
-        // "inlineSourceMap": true,               /* Emit a single file with source maps instead of having a separate file. */
-        // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
-
-        /* Experimental Options */
-        // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
-        // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-    }
+    },
+    "exclude": ["node_modules", "dist"]
```
Comment (inakianduaga) [inline tsconfig.json]: don't think we need this one with the excludes

File: index.ts (hunk 1 above)
Comment (mriceron) [reply index.ts]: Wasn't exported otherwise

**Pull request merged**